CHANGES.rst

```diff
@@ -5,7 +5,11 @@ master (unreleased)
 -------------------
 
 * Fix infinite recursion with multiple `MonitorField` and `defer()` or `only()`
-on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+  on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+* Fix issue #249 when extending `Manager` and `QuerySet` classes to add custom
```

Comment:
Thanks for updating the changelog. Can we update this to actually describe the bug that was fixed in a way that someone who has experienced it would recognize without needing to look up the issue on GitHub? E.g. something like "* Fix InheritanceManager and SoftDeletableManager to respect `self._queryset_class` instead of hardcoding the queryset class. Merge of #250, fixes #249."